internal/backend/gs/gs_test.go

```diff
@@ -100,8 +99,8 @@ func TestBackendGS(t *testing.T) {
 
 func BenchmarkBackendGS(t *testing.B) {
 	vars := []string{
+		"GOOGLE_APPLICATION_CREDENTIALS",
```

Comment:
Because we don't control the authentication now, we have to use the non-namespaced environment variable.

Reply:
Ok, fine with me.